stytch/consumer/m2m.go

```diff
@@ -0,0 +1,180 @@
+package consumer
+
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/MicahParks/keyfunc/v2"
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/stytchauth/stytch-go/v9/stytch"
+	"github.com/stytchauth/stytch-go/v9/stytch/config"
+	"github.com/stytchauth/stytch-go/v9/stytch/consumer/m2m"
+	"github.com/stytchauth/stytch-go/v9/stytch/stytcherror"
+)
+
+type M2MClient struct {
+	C       stytch.Client
+	Clients *M2MClientsClient
+	JWKS    *keyfunc.JWKS
+}
+
+func NewM2MClient(c stytch.Client) *M2MClient {
+	return &M2MClient{
+		C:       c,
+		Clients: NewM2MClientsClient(c),
+	}
+}
+
+// MANUAL(Token)(SERVICE_METHOD)
+// ADDIMPORT: "context"
+// ADDIMPORT: "fmt"
+// ADDIMPORT: "github.com/stytchauth/stytch-go/v9/stytch/config"
+// ADDIMPORT: "github.com/stytchauth/stytch-go/v9/stytch/consumer/m2m"
+// ADDIMPORT: "github.com/stytchauth/stytch-go/v9/stytch/stytcherror"
+// ADDIMPORT: "io"
+// ADDIMPORT: "net/http"
+// ADDIMPORT: "net/url"
+// ADDIMPORT: "strings"
+// ADDIMPORT: "time"
+
+// Token retrieves an access token for the given M2M Client.
+// Access tokens are JWTs signed with the project's JWKs, and are valid for one hour after issuance.
+// M2M Access tokens contain a standard set of claims as well as any custom claims generated from templates.
+func (c *M2MClient) Token(
```

Comment:
should we add in the comment from https://github.com/stytchauth/stytch-node/pull/237/files#diff-7a8ab7f1c98113d2f364e370e9324108f19e505a52505e52c485f65691c5c6fbR180 here?

Reply:
Added in https://github.com/stytchauth/stytch-go/pull/137/commits/2b005bfdb5167420f377dea78f259e032f2029ba